receptorctl/receptorctl/cli.py

```diff
@@ -51,6 +51,8 @@ def get_rc(ctx):
 def status(ctx):
     rc = get_rc(ctx)
     status = rc.simple_command("status")
+    version = status.pop('Version')
+    print(f"Version: {version}")
```

Comment:
Not sure exactly why, but it seems aesthetically unpleasing to me to put the version before the node ID.  I guess because I think of the node ID as the label for the whole node, and the other items as properties accruing to that label.  It doesn't really matter but I'd personally rather keep the node ID as the first thing printed.